webui/src/components/FailoverModal.js

```diff
@@ -265,6 +265,7 @@ class FailoverModal extends React.Component<FailoverModalProps, FailoverModalSta
         </FormField>
         <LabeledInput
           label='State provider'
+          className='meta-test__stateboardChoice'
```

Comment:
It's state provider choice, not stateboard. And provider is either stateboard or etcd2